integration-tests/tests/src/tests/objects.ts

```diff
@@ -33,7 +33,7 @@ describe("Realm objects", () => {
   describe("Interface & object literal", () => {
     it("can be created", () => {
       const realm = new Realm({ schema: [PersonSchema] });
-      let john: IPerson;
+      let john!: IPerson;
 
       realm.write(() => {
         john = realm.create<IPerson>(PersonSchema.name, {
```

Comment:
Actually, with https://github.com/realm/realm-js/pull/3937 merged, we might as well do:

```suggestion
      const john = realm.write(() => {
        return realm.create<IPerson>(PersonSchema.name, {
```

Reply:
Why are we not using classes for integration tests?  (Question is out of scope for this PR, just curious)

Reply:
Oh I had no idea #3937 was a thing, nice!